client/src/app/components/add-rule/add-rule.component.html

```diff
@@ -66,9 +66,12 @@ <h3>Parameters</h3>
                 <form #executionInterval="ngForm">
                     <h3>Run every</h3>
                     <mat-form-field class="form-element">
-                        <mat-label>Interval (minutes)</mat-label>
-                        <input [(ngModel)]="currentRule.executionInterval" matInput name="rule-interval" type="number"
-                            min="0" max="1440" step=15 required>
+                        <mat-label> Interval </mat-label>
```

Comment:
Remove spaces for consistency

Reply:
I meant the spaces around `Interval`. No other label has them and they shouldn't be there.